pilot/pkg/networking/core/v1alpha3/networkfilter.go

```diff
@@ -41,6 +41,21 @@ import (
 // redisOpTimeout is the default operation timeout for the Redis proxy filter.
 var redisOpTimeout = 5 * time.Second
 
+// buildInboundNetworkFiltersForHTTP builds the network filters that should be inserted before an HCM.
+// This should only be used with HTTP; see buildInboundNetworkFilters for TCP
+func (lb *ListenerBuilder) buildInboundNetworkFiltersForHTTP(cc inboundChainConfig) []*listener.Filter {
```

Comment:
listener_inbound is the best place to have this function instead of here may be?